frontend/src/main.jsx

```diff
@@ -81,6 +85,14 @@ const router = createBrowserRouter([
             path: "/user/favorites/",
             element: <UserFavoriteRecipe />,
           },
+          {
+            path: "/user/favorite/:id",
```

Comment:
delete this children please, the component to display favorite recipe is already created